barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/generated/AvmMini_trace.cpp

```diff
@@ -0,0 +1,346 @@
+#include <array>
```

Comment:
Could you please leave comments for each function stating what the intention of that function is? In the style like we do here: https://github.com/AztecProtocol/aztec-packages/blob/5f0e1cad2872d9d16953fa3bc8d43f6cf2709d96/barretenberg/cpp/src/barretenberg/proof_system/circuit_builder/ultra_circuit_builder.cpp#L114

If someone without context tries to read this later, they will have a tough time